src/Microsoft.Kiota.Abstractions.csproj

```diff
@@ -6,7 +6,7 @@
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <AssemblyTitle>Kiota Abstractions Library for dotnet</AssemblyTitle>
     <Authors>Microsoft</Authors>
-    <TargetFrameworks>netstandard2.0;netstandard2.1;net5.0;</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net6.0;net7.0;net8.0;</TargetFrameworks>
```

Comment:
I'm not sure what value net6/7/8 add? 
And removing net5 is a binary breaking change.
Please revert.

Reply:
I can add back `net5.0`, but I believe it would be best to include net6/7/8. This is an opinion based on the official guidance here: https://learn.microsoft.com/dotnet/standard/library-guidance/cross-platform-targeting.

Reply:
> You should explicitly target net6.0, net7.0, or net8.0 when you want to use newer .NET APIs

We're from the conditional build we have in the code, we're only targeting APIs that are available with net5 and higher. We're not targeting any API specific to net6 or higher. Hence why I don't think those additional targets are required as long at we have net5 in here.